metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -382,13 +375,10 @@ def from_entry(cls, entry: AuditLogEntry) -> "QueryEvent":
     def can_parse_exported_bigquery_audit_metadata(
         cls, row: BigQueryAuditMetadata
     ) -> bool:
-        try:
-            row["timestamp"]
-            row["protoPayload"]
-            row["metadata"]
-            return True
-        except (KeyError, TypeError):
-            return False
+        row["timestamp"]
```

Comment:
We were catching exceptions within the function in case of failures. That made it impossible to log them in our source reports. These exceptions are still being caught but outside of this function so we can log them properly.

Reply:
I'll change it so we use a different function which makes it clear what is happening